firmware/network/border_router/border_router.c

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2022 Mesh4all <mesh4all.org>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * @brief  Border Radio.
+ *
+ * @author  RocioRojas <rociorojas391@gmail.com>
+ */
+#include <stdio.h>
+#include <string.h>
+#include "border_router.h"
+
+int8_t get_wired_iface(void) {
+    int max_ifaces = gnrc_netif_numof();
+    if (max_ifaces > 0) {
+        gnrc_netif_t *iface;
+        iface = gnrc_netif_get_by_type(NETDEV_ANY, NETDEV_INDEX_ANY);
```

Comment:
you should not refers to NETDEV_ANY, in case that you are using a radio or wireless iface, you could take this iface and not a wired iface

Reply:
The module checks if it's a wired iface with:

```
netif_get_opt(iface, NETOPT_IS_WIRED, 0, &isWired, sizeof(isWired))

```

Tested with radio module with no issues